### PR TITLE
Update globals.css

### PR DESCRIPTION
**Pull request by bernco**
add z-index to .animate-progress class.
could solve the issue where the progress bar animation is appearing on top of the off-canvas element.

### CODE REVIEW

**Comment by Qu1etboy** [comment]
Firstly thank you for your contribution, but the issues are still not resolved yet. I think putting the z-index on the `.animate-progress` would not help because its position is static. Again I don't know what causes the problem I'm so confused right now 😅

File: src/components/Layout/Header.tsx [hunk 1]
```diff
@@ -18,6 +36,7 @@ export default function Header() {
           <SearchInput />
         </section>
       </header>
+      {isOpen && <OffCanvas setOpen={setOpen} />}
```
Comment (Qu1etboy) [inline src/components/Layout/Header.tsx:39]: Solved it by moving the `OffCanvas` component outside of the header and putting only the button there instead.

**Review by Qu1etboy — APPROVED** [review]
Everything looks good now. Thank you for the PR 🙏💫
Also, this is my first time seeing someone come and try to help. So I'm very happy 😆
Anyways thank you again for your help ❤️

File: src/components/Layout/Header.tsx (hunk 1 above)
Comment (bernco) [reply src/components/Layout/Header.tsx:39]: Okay great. Good luck with your project. I will be around 😊

**Pull request merged**